packages/actions-global/src/actions/show-api-error-notification.ts

```diff
@@ -1,17 +1,20 @@
 import { DOMAINS } from '@commercetools-frontend/constants';
+import { TApiErrorNotification, TApiErrorNotificationOptions } from '../types';
 import showNotification from './show-notification';
 
-export default function showApiErrorNotification({ errors, source }) {
-  return showNotification(
+export default function showApiErrorNotification({
+  errors,
+}: TApiErrorNotificationOptions) {
+  return showNotification<TApiErrorNotification>(
     {
+      id: 0,
+      domain: DOMAINS.PAGE,
       kind: 'api-error',
       values: {
-        source,
```

Comment:
I checked the codebase and I couldn't find any usage of this, so we can just ignore it.